sources/frontend.c

```diff
@@ -881,7 +881,7 @@ od_frontend_cleanup(od_client_t *client, char *context, od_status_t status)
 			assert(client->route != NULL);
 			od_frontend_error(client,
 			                  KIWI_TOO_MANY_CONNECTIONS,
-			                  "sorry, too many clients for pool");
+			                  "too many active clients for user (pool_size for user reached)");
```

Comment:
could we provide user_name and pool_size here?

Reply:
Thanks, I've addressed this issue

Reply:
LGTM